### DISCUSSION TITLE
The german Telekom mailservers - don't accept mails send from SimpleLogin

### DISCUSSION THREAD

**Discussion by Pamalosebi** [simple-login/app#686] [Feature Requests]
Sadly, Mails send through SimpleLogin do not arrive in mailboxes hosted by the german Telekom (T-Online).
They are very restrictive but should accept mails send by SimpleLogin.

They have a "rule-set" which has to be applied on a Mailserver before they accept mails of it.
-> [https://postmaster.t-online.de](https://postmaster.t-online.de)

The postmasters offer a special mailaddress if a service is (wrongly) blacklisted: postmaster@rx.t-online.de

Given reasons for being on a blacklist:
- Posting of "Backscatter"
- use of "Sender Address Verification"
- missing spam filter for outgoing e-mails
- missing or insufficient Abuse Management

**Comment by nguyenkims** [comment]
@Pamalosebi thanks for the information. Though the document at https://postmaster.t-online.de/index.en.html#t3.5 seems complete, a quick glance shows its strange decision regarding industry standards such as SPF, DKIM or popular technique as grey listing:


<img width="929" alt="image" src="https://user-images.githubusercontent.com/3167780/142211964-5b6ceb0f-ddd6-4384-aa67-5d1e08dbe8d8.png">

ISP email service sometimes have issues with properly handling emails and when that happens too often, we have no choice rather than redirecting users to a "proper" email service like ProtonMail, Tutanota, etc or even Gmail, Outlook.

**Reply by Pamalosebi** [reply]
> redirecting users to a "proper" email service

yes, I am with you. They have some "specialties".
In this case, it is used by a rather large number of people. One get one of these mailboxes with the network providers contracts.
Therefore, many just stick with it.

**Comment by Konfekt** [comment]
@nguyenkims : For T-Online to be able to unblock Simplelogin, please send postmaster@t-online.de the following information [as follows](https://postmaster-t--online-de.translate.goog/index.html?_x_tr_sl=auto&_x_tr_tl=fr&_x_tr_hl=it#t6.4):

- The IP of the Simplelogin e-mail server
- The times when it sent e-mails to T-Online (for example, today twice at 16:03)
- The rejection messages it received by t-online.de

**Reply by nguyenkims** [reply]
We have sent to this address some info, hope we'll have their responses quickly.

**Comment by SamuelBlickle** [comment]
Any news about this? I'm experiencing the same issue.

**Reply by nguyenkims** [reply]
Unfortunately we haven't heard back from t-online yet, we'll try to do a follow up with them.
